xds/internal/xdsclient/e2e_test/federation_watchers_test.go

```diff
@@ -55,21 +56,21 @@ func setupForFederationWatchersTest(t *testing.T) (*e2e.ManagementServer, string
 	t.Cleanup(serverDefaultAuthority.Stop)
 
 	// Start another management server as the other authority.
-	serverTestAuthority, err := e2e.StartManagementServer(nil)
+	serverNonDefaultAuthority, err := e2e.StartManagementServer(nil)
 	if err != nil {
 		t.Fatalf("Failed to spin up the xDS management server: %v", err)
 	}
-	t.Cleanup(serverTestAuthority.Stop)
+	t.Cleanup(serverNonDefaultAuthority.Stop)
 
-	// Create a bootstrap file in a temporary directory.
+	// Create the bootstrap
```

Comment:
"the bootstrap"? Are we a western? bootstrap file please. Or just delete this comment "bootstrapContents, err := xdsinternal.BootstrapContents" is self explanatory but I don't feel strongly about this.

Reply:
Deleted.